home-application/pom.xml

```diff
@@ -127,11 +104,23 @@
                                     <version>${revision}</version>
                                     <type>jar</type>
                                     <overWrite>true</overWrite>
-                                    <outputDirectory>${project.build.outputDirectory}/static</outputDirectory>
+                                    <fileMappers>
+                                        <org.codehaus.plexus.components.io.filemappers.FlattenFileMapper/>
+                                    </fileMappers>
+                                </artifactItem>
+                                <artifactItem>
+                                    <groupId>org.webjars.npm</groupId>
+                                    <artifactId>rapidoc</artifactId>
+                                    <version>7.2.1</version>
```

Comment:
Versions should be in a separate section

Reply:
Done.